pl_bolts/models/detection/faster_rcnn/faster_rcnn_module.py

```diff
@@ -23,7 +26,7 @@ def _evaluate_iou(target, pred):
     return box_iou(target["boxes"], pred["boxes"]).diag().mean()
 
 
-class FasterRCNN(pl.LightningModule):
+class FRCNN(pl.LightningModule):
```

Comment:
Not a strong opinion at all, but I personally prefer the class name `FasterRCNN`. If you're all okay with `FRCNN`, that's absolutely fine :] cc: @Borda

Reply:
When we have `MaskRCNN` in Bolts in the future, `MRCNN` does not sound very common...

Reply:
`FasterRCNN` collides with a torchvision import to `FasterRCNN` hence I renamed it.

Reply:
Ah, i see...

Reply:
How about renaming torchvision's FasterRCNN:
`from torchvision.models.detection.faster_rcnn import FasterRCNN as _FasterRCNN` or
`from torchvision.models.detection.faster_rcnn import FasterRCNN as torchvision_FasterRCNN`...?

Reply:
Quite possible. I'm fine with it let me rename these !